src/Console/Helper/FormHelper.php

```diff
@@ -63,7 +63,7 @@ public function interactUsingNamedForm(
 
             $submittedData = $this->formInteractor->interactWith($form, $this->getHelperSet(), $input, $output);
 
-            $form->submit($submittedData);
+            $form->submit($submittedData, false);
```

Comment:
What does `false` mean?

Reply:
When using forms through Web interfaces or Command line in interactive mode, all fields of the form are always printed out. No matter if the user fills the fields or not, all the field names/keys are going to be in the submitted data structure.

When using forms through this library in non-interactive mode, this may not be the case. 
Some fields may be optional and, when building the submitted data structure using provided command options, some fields may be missing.

The second argument of `submit` function is called `$clearMissing` and its default is `true`.
When `true`, it sets to `null` every missing field from submitted data structure.
This works ok in common scenarios but not in non-interactive mode.